test/dotnet-core-uninstall.Tests/Shared/VSVersioning/VSVersionTests.cs

```diff
@@ -26,10 +26,10 @@ public class VSVersionTests
         [InlineData(new string[] { "2.1.500", "2.1.400", "2.1.600" }, new bool[] { false, true, false })]
         [InlineData(new string[] { "2.2.100", "2.2.200" }, new bool[] { false, false })]
         [InlineData(new string[] { "2.2.100", "2.2.200", "2.2.300" }, new bool[] { false, true, false })]
-        [InlineData(new string[] { "3.0.0", "3.0.1", "5.0.0" }, new bool[] { true, true, false })]
```

Comment:
@Forgind @joeloff I think I figured out these tests now. The WindowsVersionDivisionsToExplanation dictionary is only used in ApplyWindowsVersionDivisions. This method is applying the buckets to the SDKs only. So this test was previously using runtime versioning which didn't fit into the buckets. To fix this, I modified the test to use SDK versions that fit within the buckets and the test passes now.